app/models/tenant.rb

```diff
@@ -15,6 +15,8 @@ class Tenant < ActiveRecord::Base
   has_many :owned_providers,              :foreign_key => :tenant_owner_id, :class_name => 'Provider'
   has_many :owned_ext_management_systems, :foreign_key => :tenant_owner_id, :class_name => 'ExtManagementSystem'
   has_many :owned_vm_or_templates,        :foreign_key => :tenant_owner_id, :class_name => 'VmOrTemplate'
+  has_many :owned_service_catalog_templates, :foreign_key => :tenant_owner_id, :class_name => 'ServiceTemplateCatalog'
```

Comment:
@gtanzillo 

a) oh man.
b) I have another pr that removes these and changes the ids in the tables
c) thanks for reviewing, didn't see this come in.